src/views/shared/task-list-label/__tests__/task-list-label.test.tsx

```diff
@@ -58,7 +51,7 @@ describe(TaskListLabel.name, () => {
       );
 
       expect(screen.getByText(test.text)).toBeInTheDocument();
-      expect(screen.getByTestId(`mock-tag-${test.kind}`)).toBeInTheDocument();
+      expect(container).toMatchSnapshot();
```

Comment:
does this snapshot help with coverage?
I see that the previous way of testing was good.
If you see a benifit from adding a snapshot here make sure to pass `isSnapshot` flag to the render call.

Reply:
In theory, the coverage should be the same, but running locally the snapshots have full coverage. I have not tried running coverage for the older code though, but I still feel snapshot tests are better for these files since the styles are the main differences for each kind of tag/label.

Reply:
I disagree on this, we don't want to know if any of baseweb css properties changes as this is not part of our component, we just need to test the text and the kind as it was.
Since we don't have any extra benefit from using snapshots, please change them back and limit the snapshot usage to things we discussed before.